File: java/Fibonacci.java
public class Fibonacci {
    public static void main(String[] args) {
        int n = 10;
        int[] fib = new int[n];
        fib[0] = 0;
        fib[1] = 1;
        for (int i = 2; i < n; i++) {
            fib[i] = fib[i - 1] + fib[i - 2];
        }
        for (int num : fib) {
            System.out.print(num + " ");
        }
    }
}

File: JavaScript/fibonacci.js
function fibonacci(n) {
    let fib_sequence = [0, 1];
    for (let i = 2; i < n; i++) {
        fib_sequence[i] = fib_sequence[i - 1] + fib_sequence[i - 2];
    }
    return fib_sequence;
}

console.log(fibonacci(10));
